ci/build-and-test-nupic-bindings.sh

```diff
@@ -85,16 +85,11 @@ TEST_RESULTS_DIR="${NUPIC_CORE_ROOT}/test_results"
 echo "RUNNING NUPIC BINDINGS BUILD: BUILD_TYPE=${BUILD_TYPE}, " \
      "DEST_WHEELHOUSE=${DEST_WHEELHOUSE}" >&2
 
-# Install pycapnp to get the matching capnproto headers for nupic.core build
-# NOTE Conditional pycapnp dependency should be incorporated into
-# bindings/py/requirements.txt to abstract it from upstream scripts.
-pip install pycapnp==0.5.8
```

Comment:
No longer needed since we added pycapnp to requirements.txt:
https://github.com/numenta/nupic.core/blob/ca722e9c38b19c9dc8e5b149ca079f7ecc497e44/bindings/py/requirements.txt#L6